README.md

```diff
@@ -13,11 +13,40 @@ Beyond the [table of all datasets](https://fkie-cad.github.io/COMIDDS/content/al
 We mainly focus on datasets suited for developing and evaluating methods for intrusion detection in enterprise networks, i.e., common office environments involving applications such as browsing, emailing, or text processing as well as services such as web, email, or database servers.
 We intentionally omit datasets from very different environments such as industrial control systems or Internet exchange points.
 
+## Citing this Work
+
+If you are using COMIDDS for your academic work, please cite our paper:
+<!--  {% raw %} --> 
+```
+@inproceedings{10.1145/3675741.3675754,
+  author = {Bönninghausen, Philipp and Uetz, Rafael and Henze, Martin},
+  title = {Introducing a Comprehensive, Continuous, and Collaborative Survey of Intrusion Detection Datasets},
+  year = {2024},
+  booktitle = {Proceedings of the 17th Cyber Security Experimentation and Test Workshop},
+  series = {CSET '24},
+  location = {Philadelphia, PA, USA},
+  publisher = {Association for Computing Machinery},
+  address = {New York, NY, USA},
+  url = {https://doi.org/10.1145/3675741.3675754},
+  doi = {10.1145/3675741.3675754},
+}
+```
+<!-- {% endraw %} -->
+
+If you (additionally) would like to cite specific information from within COMIDDS, we recommend to cite the release that the information is contained in, e.g.,
+<!--  {% raw %} --> 
+```
+@misc{comidds100,
+  author = {{COMIDDS} contributors},
+  title = {{COMIDDS v1.0.0 -- GitHub}},
+  year = {2024},
+  howpublished = {\url{https://github.com/fkie-cad/COMIDDS/releases/tag/v1.0.0}},
+  note = {[Online; accessed DD-MMM-YYYY]},
+}
+```
+<!-- {% endraw %} -->
+
```

Comment:
I don't know if escaping with `<!-- {% raw %} -->` is actually necessary since the README shouldn't be included in the deployment process, but it certainly doesn't hurt.